app/src/pages/inside/projectSettingsPageContainer/content/notifications/notifications.scss

```diff
@@ -15,9 +15,59 @@
  */
 
 .notifications-container {
-  margin-top: 44px;
+  margin-top: 24px;
+  width: 100%;
+  max-width: initial;
+
+  & .contentItems>div {
+    max-width: initial;
+  }
 }
 
 .button {
   margin-left: auto;
 }
+
+.name-wrapper {
+  flex: 1;
+  width: 0;
+  font-family: $FONT-ROBOTO-MEDIUM;
+  font-size: 15px;
+  line-height: 24px;
+  color: $COLOR--almost-black;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.fieldElement {
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+  height: 44px;
+}
+
+.toggle {
+  gap: 8px;
+  height: 24px;
+}
+
+.types-name {
+  text-transform: capitalize;
+}
+
+.add-rule {
+  width: 141px;
+  height: 36px;
+  padding: 7px 0px 7px 64px;
+  margin-top: 6px;
+
+  & svg > * {
```

Comment:
Does it make sense to apply the `transform` for all nested svg properties?
Will it be enough to apply it just for top-level `svg` element?